packages/ui/src/multiplying-architecture/KaotoBridge.tsx

```diff
@@ -147,11 +144,7 @@ export const KaotoBridge = forwardRef<EditorApi, PropsWithChildren<KaotoBridgePr
         <SchemasLoaderProvider>
           <CatalogLoaderProvider>
             <CatalogTilesProvider>
-              <VisibleFlowsProvider>
-                <CatalogModalProvider>
-                  <DeleteModalContextProvider>{props.children}</DeleteModalContextProvider>
-                </CatalogModalProvider>
```

Comment:
These providers already come directly from the `DesignPage`, so they are no longer needed here.